pkg/configtx/policies.go

```diff
@@ -139,13 +137,15 @@ func (c *ConfigTx) AddApplicationOrgPolicy(orgName, modPolicy, policyName string
 
 // RemoveApplicationOrgPolicy removes an existing policy from an application organization.
 // The removed policy must exist.
+// Removal will panic if application group does not exist.
```

Comment:
Will also panic if the application org does not exist when attempting to remove the policy from it